public/app/plugins/datasource/postgres/partials/query.editor.html

```diff
@@ -1,43 +1,141 @@
-<query-editor-row query-ctrl="ctrl" can-collapse="false">
-	<div class="gf-form-inline">
-		<div class="gf-form gf-form--grow">
-			<code-editor content="ctrl.target.rawSql" datasource="ctrl.datasource" on-change="ctrl.panelCtrl.refresh()" data-mode="sql">
-			</code-editor>
-		</div>
-	</div>
+<query-editor-row query-ctrl="ctrl" has-text-edit-mode="true">
```

Comment:
This file feels very generic for a SQL query editor and probably something we can create a component for later :+1: 

The exception is the help text, but should be trivial to fix.